docs/source/user-guide/configuration/settings.rst

```diff
@@ -359,7 +359,7 @@ Advanced configuration
 
 .. _disallow-soft-linking:
 
-``allow_softlinks``: Disallow soft-linking
+``allow_softlinks``: Allow soft-linking
```

Comment:
I decided to change the title too because it didn't align with the setting's name. Maybe this made more sense when we made soft links by default.